static/bidder-info/metax.yaml

```diff
@@ -0,0 +1,13 @@
+endpoint: "https://hb.metaxads.com/prebid?sid={{.PublisherID}}&adunit={{.AdUnit}}&source=prebid-server"
```

Comment:
```
curl -i --location --request POST 'https://hb.metaxads.com/prebid?sid=22&adunit=1&source=prebid-server'
curl: (6) Could not resolve host: hb.metaxads.com
```

`hb.metaxads.com` is not reachable

Reply:
hb.metaxads.com dns is mapped to our server now. this endpint is right and the prebid API will be ready in two weeks.

because publisher must be contact us first before using our adapter and the endpoint is will tested in local, i think we can move next

Reply:
@metax-kehan, let us know when prebid endpoint is ready. Accordingly, can proceed with reviews.

>  publisher must be contact us first before using our adapter 
should mention this behaviour explicitly as comments in `bidder-info/metax.yaml`

additionally create/update bidder docs PR to specify above expected behaviour

Reply:
hi @onkarvhanumante 

codes updated

1. prebid endpoint is ready now
2. comment in bidder-info/metax.yaml now
3. docs PR had the comment already

Reply:
endpoint is reachable

```
 % curl -i --location --request POST 'https://hb.metaxads.com/prebid?sid=2&adunit=33&source=prebid-server'
HTTP/1.1 401 Unauthorized
Server: nginx
Date: Mon, 05 Aug 2024 11:24:50 GMT
Content-Type: text/plain; charset=utf-8
Content-Length: 21
Connection: keep-alive

invalid sid or adunit
```